kernel_hardening_checker/checks.py

```diff
@@ -153,6 +153,11 @@ def add_kconfig_checks(l: List[ChecklistObjType], arch: str) -> None:
     l += [KconfigCheck('self_protection', 'kspp', 'STATIC_USERMODEHELPER', 'y')] # needs userspace support
     l += [KconfigCheck('self_protection', 'kspp', 'SCHED_CORE', 'y')]
     l += [KconfigCheck('self_protection', 'kspp', 'SECURITY_LOCKDOWN_LSM', 'y')]
+    l += [KconfigCheck('self_protection', 'kspp', 'LSM', '*lockdown*'),
+          OR(KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SELINUX', 'y'),
+             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_APPARMOR', 'y'),
+             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SMACK', 'y'),
```

Comment:
No, as I wrote, it's not, what we need.
Currently, we have this check:
```
    l += [OR(KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SELINUX', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_APPARMOR', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SMACK', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_TOMOYO', 'y'))] # one of major LSMs implementing MAC
```

We need a new `*value*` check, that matches it.

Reply:
![image](https://github.com/user-attachments/assets/77ea1963-1b8d-4656-bc1e-b8fb849c5112)
like this?
```Python
    l += [AND(KconfigCheck('security_policy', 'kspp', 'LSM', '*yama*'),
              major_lsm)]
    l += [AND(KconfigCheck('security_policy', 'kspp', 'LSM', '*landlock*'),
              major_lsm)]
```

Reply:
sure
```Python
    major_lsm = OR(KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SELINUX', 'y'),
                   KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_APPARMOR', 'y'),
                   KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SMACK', 'y'),
                   KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_TOMOYO', 'y'))
    l += [AND(KconfigCheck('self_protection', 'kspp', 'LSM', '*lockdown*'),
              major_lsm)]
```

Reply:
Not this, unfortunately.

We need a new `*value*` check for major LSMs:
```
    l += [OR(KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SELINUX', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_APPARMOR', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_SMACK', 'y'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'SECURITY_TOMOYO', 'y'))] # one of major LSMs implementing MAC
    l += [OR(KconfigCheck('security_policy', 'a13xp0p0v', 'LSM', '*selinux*'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'LSM', '*apparmor*'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'LSM', '*smack*'),
             KconfigCheck('security_policy', 'a13xp0p0v', 'LSM', '*tomoyo*'))]
```
OK?